### DISCUSSION TITLE
Unable to login

### DISCUSSION THREAD

**Discussion by anirbanphys** [algo2t/alphatrade#50] [General]
As mentioned, I have created the config.py with proper user ID and password information. While executing the code streaming_data.py, I am getting given issues in terminal and getting one type password from SAS alpha

Exception occurred :: [Errno 2] No such file or directory: 'access_token.txt'
Exception occurred :: [Errno 2] No such file or directory: 'access_token.txt'
Traceback (most recent call last):
  File "/home/chiru/alphatrade/examples/streaming_data.py", line 11, in <module>
    sas = AlphaTrade(login_id=config.login_id, password=config.password,
  File "/home/chiru/.local/lib/python3.10/site-packages/alphatrade/alphatrade.py", line 240, in __init__
    self.__set_access_token()
  File "/home/chiru/.local/lib/python3.10/site-packages/alphatrade/alphatrade.py", line 309, in __set_access_token
    raise requests.HTTPError(response.text)
requests.exceptions.HTTPError: {"errors":{"detail":"Internal server error"}}

Any help?

**Comment by nevatia** [comment]
Copy access_token.txt and paste where config file is

**Reply by anirbanphys** [reply]
Tried that, but still getting this

Traceback (most recent call last):
  File "/home/chiru/alphatrade/examples/sas_login_eg.py", line 24, in <module>
    sas = AlphaTrade(login_id=config.login_id, password=config.password, twofa=config.twofa)
  File "/home/chiru/.local/lib/python3.10/site-packages/alphatrade/alphatrade.py", line 240, in __init__
    self.__set_access_token()
  File "/home/chiru/.local/lib/python3.10/site-packages/alphatrade/alphatrade.py", line 309, in __set_access_token
    raise requests.HTTPError(response.text)
requests.exceptions.HTTPError: {"errors":{"detail":"Internal server error"}}

I think, now the login module should be changed, it is because it is asking one time password sent by SAS.

**Reply by nevatia** [reply]
Check twofa, modify it as per example in md file

**Reply by anirbanphys** [reply]
The problem is, they changed the two-factor authentication process, the mentioned process is no longer avaialable. I have tried before posting my issues here. What I understand is, while we run the *.py file, in terminal we should provide the password and two factor authentication

**Reply by nevatia** [reply]
Plz read doc properly. You need to enter qr code 16 digit in totp

**Reply by nevatia** [reply]
Connect us t.me/pythontrader

**Reply by anirbanphys** [reply]
Thank you for answering all of my queries, really appreciate it. 
I got your point. But the problem is, I have set the two factor authentication with google. For my case I get the 6 digit code from google authenticator, and I set the code for login for every case in the config.py file.
I tried to change the authenticator as described in the description. But not able to reset it, that option is not available in the alpha site.
